test/integration/targets/s3_bucket/aliases

```diff
@@ -0,0 +1 @@
+cloud/aws
```

Comment:
Add `posix/ci/cloud/aws` so this test will run in CI. Permissions will also need to be updated for the test to pass.